examples/python/ColumnOperations/README.md

```diff
@@ -205,13 +208,59 @@ The second function is an aggregation and sums the values in a column. We iterat
+        """
+
+        result = [_MINFLOAT]*2
+
+        # Iterate over bundled rows
+        for request_rows in request:
+            # Iterating over rows
+            for row in request_rows.rows:
+                # Retrieve the numerical value of each parameter
+                # and update the result variable if it's higher than the previously saved value
+                for i in range(0, len(row.duals)):
+                    result[i]=max(result[i], row.duals[i].numData)
+
+        # Create an iterable of dual with numerical value
+        duals = iter( [SSE.Dual(numData=r) for r in result])
+
+        # Set and send Table header
+        table = SSE.TableDescription(name='MaxOfColumns', numberOfRows=1)
+        table.fields.add(name='Max1', dataType=SSE.NUMERIC)
+        table.fields.add(name='Max2', dataType=SSE.NUMERIC)
+        md = (('qlik-tabledescription-bin', table.SerializeToString()),)
+        context.send_initial_metadata(md)
+        
+        # Yield the row data constructed
+        yield SSE.BundledRows(rows=[SSE.Row(duals=duals)])
+```
 
 ## Qlik documents
-An example document is given for Qlik Sense (SSE_Column_Operations.qvf) and QlikView (SSE_Column_Operations.qvw). There are are two sheets in this example. One contains script calls and the other contains user defined function calls. We demonstrate a tensor function, which sums two columns row-wise, and an aggregating function, which sums all rows in a column returning a single value. The data loaded in the Data Load Editor are two fields, *A* and *B*, each of which contain five numeric values.
+An example document is given for Qlik Sense (SSE_Column_Operations.qvf) and QlikView (SSE_Column_Operations.qvw). 
+There are three sheets in this example, one with script calls, one with user defined function calls 
+and one with the result of an SSE Table Load. We demonstrate a tensor function, which sums two columns row-wise, 
+and an aggregating function, which sums all rows in a column returning a single value. 
+The data loaded in the Data Load Editor are two fields, *A* and *B*, each of which contain five numeric values.
 
 The aggregating script function is called with the expression `Column.ScriptAggr('sum(args[0])', A)`, where `'sum(args[0])'` is the script and `A` is the data field. The script returns a single value when evaluated. The second script call is `Column.ScriptEval('args[0]+args[1]',A,B)` with the script `'args[0]+args[1]'` adding the two parameters `A` and `B` row-wise. The result is an array with five values, each a sum of the  corresponding values in `A` and `B`.
 
-The user defined function calls are straightforward as implemented. The aggregating function `SumOfColumn` is called with `Column.SumOfColumn(A)` and the tensor function `SumOfRows` is called with `Column.SumOfRows(A,B)`.
+The user defined function calls are straightforward to call since they become integrated in the script syntax.
```

Comment:
The user-defined functions are straightforward to call, since they become integrated in the script syntax.